rpm/SPECS/crowdsec-firewall-bouncer.spec

```diff
@@ -155,9 +156,10 @@ if command -v "$CSCLI" >/dev/null; then
 fi
 
 if [ ${START} -eq 0 ] ; then
-    echo "no api key was generated, won't start service"
+    echo "no api key was generated, won't start or enable service"
 else 
     systemctl start crowdsec-firewall-bouncer
+    systemctl enable crowdsec-firewall-bouncer
```

Comment:
Are you sure that is required ? Why is that different from crowdsec https://github.com/crowdsecurity/crowdsec/blob/master/rpm/SPECS/crowdsec.spec#L197

(Note that we only enable explicitely on fc35)